orca-core/src/main/groovy/com/netflix/spinnaker/orca/pipeline/support/DeploymentStrategyParameterSupport.groovy

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2015 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.orca.pipeline.support
+
+import com.netflix.spinnaker.orca.pipeline.model.Pipeline
+
+class DeploymentStrategyParameterSupport {
```

Comment:
Would this fit better as a method on the interface with a default noop interface implementation for Orchestration (and this implementation for Pipeline)?

`DeploymentStrategyParametersSupport.resolveStrategyParams()` vs. `stage.resolveStrategyParams()`